ui/round/src/plugins/round.nvui.ts

```diff
@@ -242,9 +242,7 @@ function createSubmitHandler(
       } else notify('Invalid move');
     }
 
-    const input = submitStoredPremove
-      ? nvui.premoveInput
-      : castlingFlavours(($input.val() as string).trim().toLowerCase());
+    const input = submitStoredPremove ? nvui.premoveInput : castlingFlavours(($input.val() as string).trim());
```

Comment:
but isn't it faster and more comfortable to allow skipping the uppercase? After this commit can we still type `nf3` instead of `Nf3`?

If not, imo we should enforce uppercase only when disambiguation requires it.

Reply:
It's case insensitive if case does not matter. First checks for a direct match, if not a lower case match https://github.com/lichess-org/lila/blob/master/ui/chess/src/sanWriter.ts#L157-L159